src/test/java/com/java/project/cucumber/RestApiSteps.java

```diff
@@ -39,4 +39,19 @@ public void verifyCityWeatherDetails(String cityName) {
         System.out.println("The wind speed: " + jsonPath.get(WIND_SPEED));
         System.out.println("The wind direction degreee: " + jsonPath.get(WIND_DIRECTION));
     }
+
+    @Then("I should receive an error message")
+    public void verifyResponseForInvalidCity() {
+        Integer statusCode = response.getStatusCode();
+        Assert.assertEquals(BAD_REQUEST_CODE, statusCode);
```

Comment:
Another approach could be: 
```suggestion
        Assert.assertEquals(BAD_REQUEST_CODE, statusCode);
```